docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.general.scss

```diff
@@ -6,3 +6,37 @@
   $size-in-rems: $size-in-pixels / $hb-root-font-size;
   @return #{$size-in-rems}rem;
 }
+
+// Returns the proper spacing based on the theme.
+// Filters through $hb-spacing-list array to find the relevant
+// spacing $size for the theme.
+@function hb-spacing-width($size: 'default', $convert-to-rem: true) {
```

Comment:
Can you add a comment about how the `$convert-to-rem` variable provides the flexibility to allow for the output of the value to be either rems or pixels? That there are a couple edge cases where pixels are necessary. Or something like that. 😄